roles/seqreports/defaults/main.yml

```diff
@@ -9,15 +9,19 @@ checkqc_bcl2fastq_output_path: Unaligned
 seqreports_data_path: "{{ sw_path }}/seqreports"
 
 seqreports_venv_name: seqreports-env
-seqreports_user: seqreports  # Only used in dev and local environment
-seqreports_user_uid: 506  # Only used in dev and local environment
-seqreports_group: 'MOLM-GG-BioInfo'  # Only used in dev and local environment
-seqreports_group_uid: 504
 
+nextflow_version: 21.04.1
 seqreports_data_fastq_screen_db_path: "{{ seqreports_data_path }}/FastQ_Screen_Genomes"
 seqreports_nxf_temp: "{{ seqreports_data_path }}/nxf_tmp/"
 seqreports_nxf_work: "{{ seqreports_data_path }}/nxf_work/"
 seqreports_nxf_logs: "{{ seqreports_data_path }}/nxf_logs/"
```

Comment:
you CAN specify these to override the defaults from the Nextflow role, but I'd argue that it shouldn't be necessary

Reply:
Perhaps we can discuss this further in future. The reason for I have separate nextflow version variable, is that I have snatched up that there are seem to be different views between us and stockholm on how early we should use new versions of nextflow. I have also a general feeling that it might make our lives easier to be able to have separate versions.